common/app/conf/switches/PerformanceSwitches.scala

```diff
@@ -5,24 +5,15 @@ import org.joda.time.LocalDate
 
 trait PerformanceSwitches {
 
-  val MinifyInlineJsSwitch = Switch(
+  val ClosureCompilerStandardOptimisation = Switch(
     "Performance",
-    "minify-inline-js",
-    "If this switch is on, InlineJs output will be minified by closure compiler",
-    safeState = Off,
+    "closure-compiler-standard",
+    "If this switch is on, the closure compiler will use standard optimisation instead of whitespace only",
```

Comment:
"If this switch is on, the inline JS will be compressed using closure compiler's standard optimisation instead of whitespace only"